installing/install_config/installing-customizing.adoc

```diff
@@ -30,19 +30,24 @@ include::modules/installation-special-config-butane-create.adoc[leveloffset=+2]
 .Additional resources
 
 * xref:../../installing/install_config/installing-customizing.adoc#installation-special-config-kmod_installing-customizing[Adding kernel modules to nodes]
-* xref:../../installing/install_config/installing-customizing.adoc#installation-special-config-storage_installing-customizing[Encrypting or mirroring disks during installation]
+* xref:../../installing/install_config/installing-customizing.adoc#installation-special-config-storage_installing-customizing[Encrypting and mirroring disks during installation]
 
 include::modules/installation-special-config-kargs.adoc[leveloffset=+1]
 ifdef::openshift-webscale[]
 include::modules/installation-special-config-rtkernel.adoc[leveloffset=+1]
 endif::openshift-webscale[]
 include::modules/installation-special-config-kmod.adoc[leveloffset=+1]
 include::modules/installation-special-config-storage.adoc[leveloffset=+1]
+
+.Additional resources
+
+* See link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/security_hardening/configuring-automated-unlocking-of-encrypted-volumes-using-policy-based-decryption_security-hardening[Configuring automated unlocking of encrypted volumes using policy-based decryption] for more information about the TPM v2 and Tang encryption modes.
+
 include::modules/installation-special-config-chrony.adoc[leveloffset=+1]
 
 == Additional resources
 
-See xref:../../installing/install_config/installing-customizing.adoc#installation-special-config-butane_installing-customizing[Creating machine configs with Butane] for information on Butane.
+* See xref:../../installing/install_config/installing-customizing.adoc#installation-special-config-butane_installing-customizing[Creating machine configs with Butane] for information on Butane.
```

Comment:
If you changed to _For more information, see_ above, same here.

Reply:
@mburke5678 thank you for your detailed peer review! I have now resolved this suggestion.